### PR TITLE
Restore build and package fake autopilot

### PR DESCRIPTION
**Pull request by arturmadrzak**
Adjust source code to build with latest MAVSDK release (0.38). Build Debian/Ubuntu package as a result and keep it as an artifact in Github.

### CODE REVIEW

**Comment by DanielePettenuzzo** [comment]
@JonasVautherin If you see some prs coming in this is because we are working on the "payload dev kit" to enable OEMs to develop their own payload manager driver. To then test the driver Artur is preparing a mavsdk based script. Since we don't want to spin up the full simulator for this I suggested to use this repo and adapt it where needed.

File: package.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		esac
+	done
+	shift "$((OPTIND - 1))"
+
+    output="${_output:-${DEF_OUTPUT}}"
+
+    dpkg-buildpackage -us -uc -nc -b
+    mkdir -p "${output}"
+    mv ../fake-autopilot* "${output}"
+}
```
Comment (QuentinVecchio) [inline package.sh:54]: indentation problem

File: .github/workflows/package.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Build, test and package
+
+on: push
```
Comment (QuentinVecchio) [inline .github/workflows/package.yml:3]: maybe better to use on `pull_request`?

**Review by QuentinVecchio — APPROVED** [review]
LGTM

File: .github/workflows/package.yml (hunk 2 above)
Comment (arturmadrzak) [reply .github/workflows/package.yml]: Yes, sure. We can limit CI to PRs, but I still opt to have master build all the time.

**Pull request merged**